pkg/apb/svc_acct.go

```diff
@@ -54,6 +55,7 @@ func (s *ServiceAccountManager) CreateApbSandbox(
 	executionContext ExecutionContext,
 	apbRole string,
 ) (string, error) {
+	metrics.SandboxCreated()
```

Comment:
Do we actually mark created before it is really created? What happens if this method returns with an error?

Reply:
nothing bad would happen, our metrics would say we created a sandbox when it actually failed to create the sandbox, I think that moving it down to once it is actually created makes sense to me.